sdk/communication/Azure.Communication.PhoneNumbers/tests.yml

```
@@ -5,6 +5,11 @@ extends:
   parameters:
     ServiceDirectory: communication
     Project: Azure.Communication.PhoneNumbers
+    MatrixConfigs:
+      - Name: PhoneNumbers_NET_matrix
+        Path: /sdk/communication/Azure.Communication.PhoneNumbers/phone-numbers-livetest-matrix.json
```

It needs to be a relative path from the repo root instead of an absolute path:

```suggestion
        Path: sdk/communication/Azure.Communication.PhoneNumbers/phone-numbers-livetest-matrix.json
```